### DISCUSSION TITLE
Displacement Time series in SBAS

### DISCUSSION THREAD

**Discussion by zzarifi** [gmtsar/gmtsar#979] [Q&A] [answered]
Hi,
I could run SBAS apparently successfully. I got Ave. Velocity and a Time Series of displacement. The first time stamp has Zero disp (,which reading Q&A I understood the reason). The second has a >0 value, but from the third time stamp all cumulative displacements for all time stamps are the same (from totally 20 time stamp, 18 have exactly the same cumulative displacement). Any idea what went wrong?

**Comment by Xiaohua-Eric-Xu** [comment]
This sounds like a problem, potentially caused by your scene.tab including a date that's not used by any of the pairs in your intf.tab. Try clean it up and let me know if that fix the problem.

**Reply by zzarifi** [reply]
Hi,
I checked the scene.tab to make sure all scenes are defined in intf.tab, then I ran the sbas again and the problem still exists. Any other suggestions?
Regards 
Zoya

**Reply by Xiaohua-Eric-Xu** [reply]
@kguns32 met this problem once, but I don't remember how we fixed the problem. So I am trying to @ her for a suggestion.

**Reply by kguns32** [reply, marked as answer]
Hi sorry for the slow response @Xiaohua-Eric-Xu @zzarifi  we have run across this before. I recently had this happen and two things I checked to fix it were (1) did the time series network have enough connections over the problematic date (e.g., you said everything after the third time stamp was the same, so check the third time stamp date in your baseline plot and see if there are enough pair connections); and (2) I think we have applied a very small smoothing parameter to just stabilize the inversion (e.g., set smooth 0.1 in SBAS).
